pennylane/optimize/qng.py

```diff
@@ -86,6 +87,53 @@ class QNGOptimizer(GradientDescentOptimizer):
         * For multi-QNode models, we don't know what geometry is appropriate
           if a parameter is shared amongst several QNodes.
 
+        If the objective function is VQE/VQE-like, i.e. a function of a group
+        of QNodes that share an ansatz, there are two ways to use the optimizer:
+
+        * Realize the objective function as a VQECost object, which has
+          the metric_tensor function.
+
+        * Manually provide the metric_tensor_fn as the metric_tensor of
+          one of the QNodes in the group of QNodes involved in the objective function.
+
+    **Examples:**
+
+    For VQE/VQE-like problems, the objective function for the optimizer can be
+    realized as a VQECost object.
+
+    >>> dev = qml.device("default.qubit", wires=1)
+    >>> def circuit(params, wires=0):
+    >>>     qml.RX(params[0], wires=wires)
+    >>>     qml.RY(params[1], wires=wires)
+    >>> coeffs = [1, 1]
+    >>> obs = [qml.PauliX(0), qml.PauliZ(0)]
+    >>> H = qml.Hamiltonian(coeffs, obs)
+    >>> cost_fn = qml.VQECost(circuit, H, dev)
+    >>> eta = 0.01
+    >>> init_params = [0.011, 0.012]
+    >>> opt = qml.QNGOptimizer(eta)
+    >>> theta_new = opt.step(cost_fn, init_params)
+    >>> print(theta_new)
+    [0.011445239214543481, -0.027519522461477233]
+
+    Alternatively, the same objective function can be used for the optimizer
+    by manually providing the metric_tensor_fn.
+
+    >>> dev = qml.device("default.qubit", wires=1)
+    >>> def circuit(params, wires=0):
+    >>>     qml.RX(params[0], wires=wires)
+    >>>     qml.RY(params[1], wires=wires)
+    >>> coeffs = [1, 1]
+    >>> obs = [qml.PauliX(0), qml.PauliZ(0)]
```

Comment:
If this is the same as above, I suppose it can be removed?

Reply:
Good point! I'll remove them.